src/views/UserProfileView.vue

```diff
@@ -268,14 +265,20 @@
   @import "../styles/base";
 
   .profile__header{
-    padding: .25rem 1rem;
-    background-color: $actions_ribbon_background_color;
-    .address{
-      color: $standard_font_color;
-    }
-    .count{
-      color: $light_font_color;
-      font-size: .6rem;
+    top: 0;
+    position: sticky;
+    position: -webkit-sticky;
+    z-index: 10;
```

Comment:
why z-index?

Reply:
Because Position sticky is fixed when scrolled and it goes under the content.